<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YasukawaMotoComClass</class>
 <widget class="QMainWindow" name="YasukawaMotoComClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YasukawaMotoCom</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonServoON">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Servo ON</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonHoldON">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold ON</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGetStatus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGetStatus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetPos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>351</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Pulse</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Torque</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Robot</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rem Torque</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetMovPos">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>131</width>
      <height>451</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Robot</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonMovRel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mov rel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxServoON">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ServoON</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGetStatus_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxVelocity">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelGetStatus_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mm/s</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonPushMove">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Push move</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonRecordMotion">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Revord motion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGetStatus_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Torque mergin</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxTorqueMergin">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <double>4.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>739</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>801</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
